sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/TokenRefreshOptions.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.common;
+
+/**
+ * Options for refreshing CommunicationTokenCredential
+ */
+public class TokenRefreshOptions {
```

Comment:
To avoid naming ambiguity I can hear Kryzsztof ask to rename this to `CommunicationTokenRefreshOptions` and I think we should. :-)

Reply:
Done ;)